### DISCUSSION TITLE
Brew install (instructions) no longer accurate?

### DISCUSSION THREAD

**Discussion by stevenbarich** [The-Wineskin-Project/WineskinServer#489] [General]
Hello,
I am attempting to install Wineskin on OS Ventura (Nov 5, 2023).  The instructions on this site are pretty simple using homebrew: "brew install --no-quarantine gcenx/wine/unofficial-wineskin"

However, running this command line produces an error:
"Warning: No available formula or cask with the name "gcenx/wine/unofficial-wineskin". Did you mean gcenx/wine/cx-llvm?"

So, I then ran an edited command of:
"brew install --no-quarantine gcenx/wine/cx-llvm"

That seemed to work better...stuff installed.

However, there were more Caveats:
"==> Caveats
cx-llvm is keg-only, which means it was not symlinked into /usr/local,
because this is an alternate version of another formula.

If you need to have cx-llvm first in your PATH, run:
  echo 'export PATH="/usr/local/opt/cx-llvm/bin:$PATH"' >> ~/.profile

For compilers to find cx-llvm you may need to set:
  export LDFLAGS="-L/usr/local/opt/cx-llvm/lib"
  export CPPFLAGS="-I/usr/local/opt/cx-llvm/include"
==> Summary
🍺  /usr/local/Cellar/cx-llvm/22.1.1: 2,246 files, 998.2MB"

Should I be at all concerned here?  
But more importantly, shouldn't the instructions for install using homebrew be updated?

Cheers,
Steve

**Comment by Gcenx** [comment]
_Where_ exactly are you getting these instructions as the README.md file is correct.

**Reply by stevenbarich** [reply]
Hello,
This was returned to me in Terminal.

**Reply by stevenbarich** [reply]
Wait...maybe you meant the install instructions?
It is on the Wiki tab on this site.

**Comment by stevenbarich** [comment]
![image](https://github.com/Gcenx/WineskinServer/assets/122837171/cfd1c3f3-767f-469d-88d0-7dcc4edd05b2)

**Reply by Gcenx** [reply]
I’m surprised you’d went directly to the wiki over the README.md file, but I’ve now corrected that wiki page also.

**Reply by stevenbarich** [reply]
I'm surprised as well!  lol.  Thanks.
